yorkie/src/main/kotlin/dev/yorkie/core/Client.kt

```diff
@@ -290,14 +291,14 @@ public class Client @VisibleForTesting internal constructor(
                     attachment.document.key.documentBasedRequestHeader,
                 ).also {
                     latestStream = it
+                    attachment.document.publishEvent(StreamConnectionChanged.Connected)
                 }
                 val streamJob = launch(start = CoroutineStart.UNDISPATCHED) {
                     val channel = stream.responseChannel()
                     var retry = 0
                     while (!stream.isReceiveClosed() && !channel.isClosedForReceive) {
                         val receiveResult = channel.receiveCatching()
                         receiveResult.onSuccess {
-                            attachment.document.publishEvent(StreamConnectionChange.Connected)
```

Comment:
don't we need this line?
if the stream fails then succeed on retry, users would not receive the connected state.

Reply:
I missed it. fixed in [4f46596](https://github.com/yorkie-team/yorkie-android-sdk/pull/181/commits/4f465969d8507f4916feeaeec7e84af1603792aa)